doc/user-guide.md

```diff
@@ -90,6 +90,13 @@ Replace the default handler with the reference [memcached handler][memcached_han
 
 ### Build and run the operator
 
+There is 2 ways to run the operator:
+
+- As pod inside Kubernetes cluster
+- As go program outside cluster
+
+#### 1. Run as pod inside Kubernetes cluster
+
```

Comment:
Probably mention that this is the suggested production setup, since we mention the out-cluster one is for dev and test.

Reply:
done